src/main/java/org/team1540/robot2019/drivecontrol/commands/SimplePointToVisionTarget.java

```diff
@@ -0,0 +1,115 @@
+import org.team1540.robot2019.Robot;
+import org.team1540.robot2019.Tuning;
+import org.team1540.robot2019.datastructures.threed.Transform3D;
+import org.team1540.robot2019.datastructures.twod.Twist2D;
+import org.team1540.robot2019.datastructures.utils.TrigUtils;
+import org.team1540.robot2019.utils.ControlUtils;
+import org.team1540.robot2019.utils.TankDriveTwist2DInput;
+import org.team1540.rooster.drive.pipeline.FeedForwardProcessor;
+import org.team1540.rooster.drive.pipeline.UnitScaler;
+import org.team1540.rooster.functional.Executable;
+
+public class SimplePointToVisionTarget extends PIDCommand {
+
+    public static final Logger logger = Logger.getLogger(SimplePointToVisionTarget.class);
+
+    // Max/Min angular velocity
+    private static final double MIN_VEL_THETA = 0.1;
+    private static final double MAX_VEL_THETA = 1.8;
+
+    // Constants for angular VPID controller
+    private static final double ANGULAR_KP = -6;
+    private static final double ANGULAR_KI = 0;
+    private static final double ANGULAR_KD = -10;
+
+    private static final double ANGLE_OFFSET = Math.toRadians(5.5); // Degrees offset from center of target
+
+    // Goal tolerances for angle
+    private static final double GOAL_TOLERANCE_ANGULAR_POSITION = Math.toRadians(0.6);
+    private static final double GOAL_TOLERANCE_ANGULAR_VELOCITY = 0.3;
+
+    private Executable pipeline;
+    private TankDriveTwist2DInput twist2DInput;
+    private Double goal = null;
+
+    public SimplePointToVisionTarget() {
+        super(ANGULAR_KP, ANGULAR_KI, ANGULAR_KD);
+        requires(Robot.drivetrain);
+        twist2DInput = new TankDriveTwist2DInput(Tuning.drivetrainRadiusMeters);
+        pipeline = twist2DInput
+            .then(new FeedForwardProcessor(0, 0, 0))
+            .then(new UnitScaler(Tuning.drivetrainTicksPerMeter, 10))
+            .then(Robot.drivetrain.getPipelineOutput());
+    }
+
+    @Override
+    protected void initialize() {
+        double x = Math.toRadians(NetworkTableInstance.getDefault().getTable("limelight-a").getEntry("tx").getDouble(0));
+        if (x == 0) {
+            Transform3D prevGoal = Robot.lastOdomToVisionTarget;
+            if (prevGoal == null) {
+                Robot.drivetrain.stop();
+                logger.warn("Unable to find target and no alternative specified! Ending...");
+                return;
+            }
+            goal = prevGoal.toTransform2D().getTheta();
+            logger.info("Unable to find target. Using alternative goal angle: " + goal);
+        } else {
+            goal = x - Hardware.navx.getYawRadians();
+            logger.info("Point lineup simple starting. Initial goal angle: " + goal);
+        }
+        Robot.drivetrain.configTalonsForVelocity();
+    }
+
+    @Override
+    protected void execute() {
+        if (goal == null) {
+            return;
+        }
+        double x = Math.toRadians(NetworkTableInstance.getDefault().getTable("limelight-a").getEntry("tx").getDouble(0));
```

Comment:
You seem to have a wrapper some places and not others

Reply:
Done.